modules/json/src/lib/parse-json-in-batches.js

```diff
@@ -65,21 +66,21 @@ export default async function* parseJSONInBatches(asyncIterator, options) {
   }
 
   // yield final batch
-  const jsonpath = parser.getJsonPath().toString();
+  const jsonpath = parser.getStreamingJsonPath();
```

Comment:
@ibgreen please note that this was calling `getJsonPath()` as opposed to `getStreamingJsonPath()` like the rest of the batch types. Let me know if this fix isn't correct.

Reply:
Yes looks correct.